README.md

```diff
@@ -619,8 +619,8 @@ will output
 
 An accessible way to add button links into content, that can also allow cross domain tracking with [Google Analytics](https://support.google.com/analytics/answer/7372977?hl=en)
 
-This button component is [extended from static](http://govuk-static.herokuapp.com/component-guide/button) for [use in govspeak](http://govuk-static.herokuapp.com/component-guide/govspeak/button)
-Note: Ideally we'd use the original component directly but this currently isnt possible
+This button component mimics the component from the [components gem](https://components.publishing.service.gov.uk/component-guide/button) for use in govspeak.
+Note: Ideally we'd use the original component directly but this currently isn't possible.
```

Comment:
Can you add why it's not possible please? Maybe in the Description of the PR itself.

Reply:
Have reworked the code to make this line untrue, and removed it and modified the line above.